### PR TITLE
add form prefill functionality from log data

### PR DESCRIPTION
**Pull request by aleks-and**

### CODE REVIEW

File: src/js/utils/getHexFromNumber.js [hunk 1]
```diff
@@ -1,5 +1,11 @@
-export default number => {
-    const hex = number.toString(16);
+export default ( number, withPrefix = true, withSpace = false ) => {
+    let hex = number.toString(16);
 
-    return `0x${hex.length % 2 === 0 ? hex : `0${hex}`}`;
+    hex = hex.length % 2 === 0 ? hex : `0${hex}`;
+
+    if ( withSpace ) {
+        hex = hex.match(/.{2}/g).join(' ');
```
Comment (DarkPark) [inline src/js/utils/getHexFromNumber.js:7]: we can get here something like this `0x07 5b c7 c8`

Comment (aleks-and) [reply src/js/utils/getHexFromNumber.js:7]: fixed as discussed

**Review by DarkPark — APPROVED** [review]

File: src/js/constants/codecBuildDefaults.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import {accessLevels} from 'jooby-codec/mtx/constants/index.js';
```
Comment (DeadByElpy) [inline src/js/constants/codecBuildDefaults.js:1]: mtxDefaults.js?

File: src/js/contexts/CodecBuildPrefillDataContext.jsx [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+                parameters: JSON.stringify(command.command.parameters, null, 4),
+                id: uuidv4()
+            }));
+
+            const hardwareType = log.hardwareType
+                ? commandTypeConfigMap[log.commandType].hardwareTypeList.find(({value}) => value === log.hardwareType)
+                : null;
+
+            const parameters = {
+                accessLevel: isUndefined(accessLevel) ? codecBuildDefaults.accessLevel : accessLevel,
```
Comment (DeadByElpy) [inline src/js/contexts/CodecBuildPrefillDataContext.jsx:45]: need to discuss

**Review by DeadByElpy — CHANGES_REQUESTED** [review]

**Pull request merged**